### ISSUE TITLE
Learning parameters of each simulated device

### ISSUE THREAD

**Issue by pentium3** [tensorflow/federated#798]
Does tensorflow-federated support assigning different hyper-parameters(like batch-size or learning rate) for different simulated devices?

**Comment by ZacharyGarrett** [comment]
This doesn't seem like a bug/feature request for the code, but rather usage question. This might be better asked at https://stackoverflow.com/questions/tagged/tensorflow-federated. Could you ask there?
